### DISCUSSION TITLE
Alternative to ButtonType

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3135] [Q&A] [answered]
in 2.1x, there was a prop for Button called ButtonType. And we would just import ButtonType in bentleyui-core and assign it like buttonType = {ButtonType.blue}. This was a simpler implementation for button as it took out the need to worry about Css, hovering, etc. Now in 3.0, this prop has been removed. I am aware that there is an option StyleType, and we could use cta, borderless, etc. But this only provides limited functionality and i have to implement my css from the scratch and assign it to style everytime i make a button. 

Is there any alternatives for ButtonType that would be just plug and play, so that there would be one less thing to worry about? If not, then what is the recommended way for implementing Button, such as making a wrapper for button such that we wouldnt need to assign the similar arguments each time we make a button?

**Comment by bsteinbk** [comment, marked as answer]
@hanan-commits-work - The Button styleType of "high-visibility", in the itwinui-react button, will result in a blue button and react to hover effects. What functionality are you missing that you want to add via a wrapper?

Adding @veekeys from itwinui team.

**Reply by hanan-commits-work** [reply]
Just the basic functionality for button. But I do not want to assign multiple parameters to achieve the same functionality as 2.1x without the extra setups.

**Reply by hanan-commits-work** [reply]
Let me try default button with high-visibility

**Reply by veekeys** [reply]
Hey @hanan-commits-work 
In v3 some of the core components were deprecated in favor of using iTwinUI, which is next generation of core UI components. To have the same blue button, it requires the same amount of code
`<Button styleType='high-visibility'>My Button</Button>`.
Be sure to import `Button` component from `@itwin/itwinui-react` package.
You can find the same example (and a lot more) in our demo page:
https://itwin.github.io/iTwinUI-react/?path=/story/buttons-button--high-visibility where code for the component is also visible in the section below.
Let me know if you need some more information or some functionality is missing.

**Reply by hanan-commits-work** [reply]
Thank You. This is exactly what I needed.
